src/equiv.js

```diff
@@ -59,6 +59,10 @@ QUnit.equiv = (function() {
 		return false;
 	}
 
+	function getRegExpFlags( regexp ) {
+		return regexp.flags || regexp.toString().match(/[gimuy]*$/)[0];
```

Comment:
You can make JSHint exceptions, but it's probably better to define the regular expressions by constructor (e.g., `new RegExp( "foo", "u" )`).